source/community.html.erb

```diff
@@ -19,7 +19,7 @@ responsive: true
   </p>
 
   <p>
-    <a href="https://discordapp.com/invite/zT3asNS">Ember Community Discord</a> &mdash; use <a href="https://discordapp.com/invite/zT3asNS">this link</a> to receive an invitation.
+    <a href="https://discordapp.com/invite/zT3asNS">Ember Community Discord</a> &mdash; use <a href="https://discordapp.com/invite/zT3asNS">this link</a> to receive an invitation. (Looking for the Ember Community Slack? We recently switched away from Slack to [Discord](https://discordapp.com). Read more about why [here](https://github.com/emberjs/rfcs/pull/345)).
```

Comment:
As @melsumner suggested we could probably remove the whole invite part, as Discord now allows users to join the chat directly, instead of receiving an invite (as it was the case with Slack before). This could e.g. only say

```
    <a href="https://discordapp.com/invite/zT3asNS">Ember Community Discord</a> &mdash; ask questions and chat with community members in real-time.(Looking for the Ember Community Slack? We recently switched away from Slack to [Discord](https://discordapp.com). Read more about why [here](https://github.com/emberjs/rfcs/pull/345)).
```

What do you think?